### ISSUE TITLE
 I can not execute debugging

### ISSUE THREAD

**Issue by tokufxug** [magic-script/magic-script-cli#42]
If installed with magic-script build \ [-i] it should be installed with debug version, but debug is set to false.

The following error occurs when executing magic-script run pacakge name.

exec error: Error: Command failed: mldb launch --auto-net-privs <package name>
Can not set --auto-net-priv for non-debuggable application: <package name>

![Capture_000058](https://user-images.githubusercontent.com/323081/55557950-0be4e700-56b9-11e9-84ab-1eeb8be1b4a1.JPG)

Windows 10 Version 1803(OS build 17134.648)
Lumin OS v0.95.0
Lumin SDK v0.20.0
node v10.15.3
npm 6.4.1
magic-script 1.0.3

Sadao Tokuyama
https://twitter.com/tokufxug

**Comment by kpiascik** [comment]
@tokufxug If you look at the top of your screenshot there is a log about "Adding tail data".  This indicates that debuggable tail data was added to the mpk.  You should be able to launch the the app from the icon grid and you can also run "magic-script run --debug=false".  I'm investigating on my side to see what might be a possible root cause.
If you're ok with sharing you mpk then that would also be useful.

**Comment by tokufxug** [comment]
Upload mpk and source set.
[hello-leaper.zip](https://github.com/magic-script/magic-script-cli/files/3050183/hello-leaper.zip)

**Comment by kpiascik** [comment]
@tokufxug OK, I looked at your code and digest file and the digest file appears to not have tail data added.  I don't know why this could happen but I was not able to reproduce it on my Windows 10 installation.  I was able to re-build and sign your application without issue.
Does this happen for all MagicScript apps you're building or just this one?
My only guess at this point could be the localization of your Windows.
Can you please try building from withing MINGW or Git Bash?  Does that help?

**Comment by tokufxug** [comment]
_Does this happen for all MagicScript apps you're building or just this one?_
All app can not be debugged.

_Can you please try building from withing MINGW or Git Bash? Does that help?_
I got an error in the same way.

Launching: net.magicmodelers.helloleaper
exec error: Error: Command failed: mldb launch --auto-net-privs net.magicmodelers.helloleaper
Cannot set --auto-net-priv for non-debuggable application: net.magicmodelers.helloleaper

Failed to launch: net.magicmodelers.helloleaper

**Comment by tokufxug** [comment]
The cause has been found.

An error occurred when running taildata_v3.py in mxs-sign.js.
_let tailDataCommand = `${python} ${script} --sbox USER --debuggable ${DIGEST_PATH}`;_

The python and script variables contain the **mlsdkRoot** path.
My **mlsdkRoot** contains the username. This is because mpk was generated without the taildata being created because the **user name contained half-width space**.

Also, although an error has occurred, in the case of console.error, the error information has not been output to the console, so it appears to have ended normally.

magic-script-cli/util/[mxs-sign.js](https://github.com/magic-script/magic-script-cli/blob/master/util/mxs-sign.js) (169 - 181)

Current
```
let python = path.join(mlsdkRoot,
    process.platform === 'win32'
      ? '/tools/python3/python.exe'
      : '/tools/python3/bin/python3.5'
  );
let script = path.join(mlsdkRoot, '/tools/mabu/src/taildata_v3.py');
  let tailDataCommand = `${python} ${script} --sbox USER --debuggable ${DIGEST_PATH}`;
  console.info('Adding tail data');
  exec(tailDataCommand, (err, stdout, stderr) => {
    console.log(stdout);
    if (err) {
      console.error('Error Adding tail data:', err);
    }
    if (stderr) {
      console.error('Error Adding tail data:', err);
    }
    signFile();
  });
}
```
correction
```
// 1. correction
let python = path.join('\"' + mlsdkRoot + '\"',
    process.platform === 'win32'
      ? '/tools/python3/python.exe'
      : '/tools/python3/bin/python3.5'
  );
  // 2. correction
  let script = path.join('\"' + mlsdkRoot + '\"', '/tools/mabu/src/taildata_v3.py');
  let tailDataCommand = `${python} ${script} --sbox USER --debuggable ${DIGEST_PATH}`;
  console.info('Adding tail data');
  exec(tailDataCommand, (err, stdout, stderr) => {
    if (err) {
      // 3. correction
      process.stderr.write('Error Adding tail data:', err);
    }
    if (stderr) {
      // 4. correction
      process.stderr.write('Error Adding tail data:', stderr);
    }
    signFile();
  });
}
```
By fixing it with the "correction" version, the "magic-script run" no longer throws an error.

**Comment by kpiascik** [comment]
I've updated the source code with changes similar to what you suggested above.  Please take a look and let me know if this resolves your issue.  Note it's not published to npm yet, but should be soon.

**Comment by tokufxug** [comment]
I confirmed the correction of the source.
No problem.

I reflected the modified source in my magic-script-cli myself. After that, I ran build and debug.
The debug execution also worked without problems.

I appreciate your kind response during your busy time.

**Comment by kpiascik** [comment]
fixed with #66
